lib/timer.py

```diff
@@ -117,5 +116,8 @@ def run_jobs(cfg: UAConfig, current_time: datetime):
 if __name__ == "__main__":
     cfg = UAConfig()
     current_time = datetime.utcnow()
-    setup_logging(logging.INFO, logging.DEBUG)
+
+    setup_logging(logging.CRITICAL, logging.DEBUG, logger=LOG)
```

Comment:
Since we are going to have two distinct LOG accessing the same file, can't we have that race condition issue here ?

Reply:
Good question, I had the same doubt and went through it: we will not.
In the case that a different process has a logger logging to the same file (for instance, timer.py runs exactly at the same time as you type a ua command), the loggers will not write the file at the same time.
Worse that can happen is that the lines will be interspersed.

Reply:
We decided to use a separate file for the timer, avoiding any risk